src/platforms/esp32/components/avm_builtins/network_driver.c

```diff
@@ -761,6 +763,32 @@ static void start_network(Context *ctx, term pid, term ref, term config)
     port_send_reply(ctx, pid, ref, OK_ATOM);
 }
 
+static void get_sta_rssi(Context *ctx, term pid, term ref)
+{
+    int sta_rssi = -999;
```

Comment:
Indeed. That makes sense, I had set it to 0 originally, but changed that to an impossible value during early testing to be sure the value was being changed, because 0 being returned is an indication of error, even if the actual error returned is ESP_OK.